src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -1269,6 +1269,30 @@ inline void rpp_load16_u8_to_u32_avx(Rpp8u *srcPtr, __m256i *p)
     p[1] = _mm256_setr_m128i(_mm_shuffle_epi8(px, xmm_pxMask08To11), _mm_shuffle_epi8(px, xmm_pxMask12To15));    /* Contains pixels 09-16 */
 }
 
+inline void rpp_load32_u8_avx(Rpp8u *srcPtr, __m256i *p)
```

Comment:
I wouldn't suggest doing inline function for one line code where you can use the _mm_256_load_xxx directly

Reply:
Sure will modify this